NuGet.config

```diff
@@ -3,5 +3,9 @@
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
Sure, I will remove these feeds from the nuget.config. 
Is the mirroring work tracked in an issue that I can watch?